### PR TITLE
New Rule: VK open redirect

### PR DESCRIPTION
**Pull request by aidenmitchell**

### CODE REVIEW

File: detection-rules/open_redirect_vk.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: "Open redirect: VK"
+description: |
+  Message contains use of the VK open redirect, but the sender is not VK. This has been exploited in the wild.
+type: "rule"
```
Comment (jkamdjou) [inline detection-rules/open_redirect_vk.yml:4]:
```suggestion
type: "rule"
authors:
  - twitter: "vector_sec"
```
https://sublimecommunity.slack.com/archives/C02N1F7FP7C/p1686829879572459

Comment (aidenmitchell) [reply detection-rules/open_redirect_vk.yml:4]: perfect, I wasn't sure what his handle was and was going to ping him in Slack, thanks Josh!

**Comment by morriscode** [comment]
Testing on this looks solid!

**Pull request merged**